hedera-node/src/main/java/com/hedera/services/store/contracts/ContractsStore.java

```diff
@@ -176,9 +181,21 @@ public void commit() {
 			blobStorageSource.put(address.toArray(), code.toArray());
 		});
 
+		provisionalAccountDeletes.forEach(address -> ledger.destroy(EntityIdUtils.accountParsedFromSolidityAddress(address.toArray())));
+
+		provisionalAccountStorageClearings.forEach(address -> {
+			//todo somehow obtain the AccountStorageMap for that Address
+			//todo for each entry (is it always 1?) in the map see how to clear ContractLeafStore and ContractHashStore -> release()?
+			//todo is that enough?
+
+			
+		});
```

Comment:
We have to discuss it with Richard and Jasper -> we must be able to remove/clear all of the storage for a given account. They might not have prepared for that